templates/python-crawlee-beautifulsoup/requirements.txt

```diff
@@ -2,4 +2,4 @@
 # https://pip.pypa.io/en/latest/reference/requirements-file-format/
 
 apify < 3.0
-crawlee[beautifulsoup] < 0.6
```

Comment:
Just a thought - couldn't we use `*` instead? The SDK should limit it to the newest compatible version already

Reply:
Makes sense, thanks.